change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Fixes #6884",
```

This comment field is used to generate changelog entries in [CHANGELOG.md](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/CHANGELOG.md). It defaults to the chosen commit message, but a more descriptive comment would be useful here:

```suggestion
  "comment": "update picker component roles",
```